brjs-core/src/main/java/org/bladerunnerjs/model/engine/AbstractRootNode.java

```diff
@@ -89,7 +89,7 @@ public void registerNode(Node node) throws NodeAlreadyRegisteredException
 	public void clearRegisteredNode(Node node) {
 		String normalizedPath = node.dir().getAbsolutePath();
 		List<Node> nodesForPath = nodeCache.get(normalizedPath);
-		nodesForPath.remove(node.getTypeName());
+		nodesForPath.remove(node);
```

Comment:
No. Seems to be an arbitrary change where nothing broke before and nothing breaks now. So why do we need the change? Or is that line even needed if it never worked....?